mu4/appshell/qml/HomePage/AddonsContent.qml

```diff
@@ -105,15 +106,16 @@ Rectangle {
                 }
             }
         }
-
-        Rectangle {
+        
+        LanguagesContent {
             id: languagesComp
-            color: ui.theme.backgroundColor
-            StyledTextLabel {
-                anchors.fill: parent
-                verticalAlignment: Text.AlignVCenter
-                horizontalAlignment: Text.AlignHCenter
-                text: "Languages Module"
+
+            Connections {
```

Comment:
But it seems that you can use it in simple property binding

`LanguagesContent {
    search: searchField.searchText
}`

Reply:
fixed